src/config/config.js

```diff
@@ -0,0 +1,56 @@
+import collector from '../collector';
```

Comment:
I'd suggest renaming this file "inline" or something analogous

Reply:
But it isn't inline? It's the config object loader 
(see comments in class).

Reply:
The comment refers to `Agent instantiation object`, which is what this is, correct? If so, I'd suggest renaming to something that matches that a little better. The file path of `config/config.js` just seems a little confusing to me personally.